### PR TITLE
31 23 admin dashboard invoices sorted by least recent

### LINKED ISSUE #31: 23. Admin Dashboard Invoices sorted by least recent
As an admin,
When I visit the admin dashboard (/admin)
In the section for "Incomplete Invoices",
Next to each invoice id I see the date that the invoice was created
And I see the date formatted like "Monday, July 18, 2019"
And I see that the list is ordered from oldest to newest

### PR DESCRIPTION
**Pull request by Daniel-Gallagher92**
__x__ Wrote Tests __x__ Implemented __x__ Reviewed

Neccesary checkmarks:

    [x] All Tests are Passing

    [x] The code will run locally

Type of change

    [x] New feature
    [x] Bug Fix

Implements/Fixes:

    description closes #31 

Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

Testing Changes

    [] No Tests have been changed
    [] Some Tests have been changed
    [x] All of the Tests have been changed(Please describe what in the world happened)

    I added before each block and made tests for US_22 more robust 

Checklist:

    [x] My code has no unused/commented out code
    [x] I have reviewed my code
    [x] I have commented my code, particularly in hard-to-understand areas
    [x] I have fully tested my code

(REQUIRED)Please Include a link to a gif of your feelings about this branch

Link: https://media.giphy.com/media/xoHntNXFYkfzGAftEv/giphy.gif

### CODE REVIEW

File: spec/features/admin/dashboard_spec.rb [hunk 1]
```diff
@@ -153,18 +150,43 @@
...
+
+  it "displays each incomplete invoice, oldest to newest" do 
+
+    visit admin_index_path
+    
+    within ".incomplete-invoices" do 
+      expect(page).to_not have_content(@invoice_9.id)
+    end
+
+    within("#invoice-#{@invoice_1.id}") do
```
Comment (z-fitch) [inline spec/features/admin/dashboard_spec.rb:176]: very nice, good use of within blocks here

**Review by z-fitch — APPROVED** [review]
looks good

File: app/views/admin/index.html.erb [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 <div class="column">
   <section class="top_customers">
-    <h3>  Top 5 Customers</h3>
+    <h3>  Top 5 Customers </h3>
```
Comment (ethanrossblack) [inline app/views/admin/index.html.erb]: Just curious, what's the reason that you're leaving spaces around the heading text?

File: app/views/admin/index.html.erb [hunk 3]
```diff
@@ -13,7 +13,7 @@
...
       <ol>
         <% @top_5_customers.each do |customer| %>
           <li><%="#{customer.first_name} #{customer.last_name} has made #{customer.result_count} successful transactions"%></li><br>
```
Comment (ethanrossblack) [inline app/views/admin/index.html.erb]: Is there a chance we can change the variable name `result_count` to something more intuitive? Maybe `successful_transactions`?

File: spec/models/invoice_spec.rb [hunk 4]
```diff
@@ -12,14 +122,23 @@
...
-  describe "class methods" do
-    it "can find incomplete invoices" do
-      customer_1 = Customer.create!(first_name: "Joey", last_name: "Smith")
-      customer_2 = Customer.create!(first_name: "Cecilia", last_name: "Jones")
-      invoice_1 = Invoice.create!(status: 0, customer: customer_1)  
-      invoice_2 = Invoice.create!(status: 1, customer: customer_2)
 
-      expect(Invoice.incomplete_invoices).to eq([invoice_1])
-    end
+  it "Can return all invoices with unshipped items ordered by oldest to newest" do
```
Comment (ethanrossblack) [inline spec/models/invoice_spec.rb]: did you want to put this inside a 
```
describe "instance methods" do
```
block?

File: spec/models/invoice_spec.rb [hunk 5]
```diff
@@ -12,14 +122,23 @@
...
+  # describe "class methods" do
+  #   it "can find incomplete invoices" do
+  #     customer_1 = Customer.create!(first_name: "Joey", last_name: "Smith")
+  #     customer_2 = Customer.create!(first_name: "Cecilia", last_name: "Jones")
+  #     invoice_1 = Invoice.create!(status: 0, customer: customer_1)  
+  #     invoice_2 = Invoice.create!(status: 1, customer: customer_2)
+
+  #     expect(Invoice.incomplete_invoices).to eq([invoice_1])
+  #   end
+  # end
```
Comment (ethanrossblack) [inline spec/models/invoice_spec.rb]: what's this from?

**Review by ethanrossblack — APPROVED** [review]
I have a few minor suggestions, but otherwise looks good

**Pull request merged**